web_many2one_simple/README.rst

```diff
@@ -0,0 +1,124 @@
+===================
+Web Many2One Simple
+===================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/13.0/web_many2one_simple
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-13-0/web-13-0-web_many2one_simple
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a new widget to many2one fields that allows to link records
+without any extra features (no autocomplete and no partial search).
+
+* Preview
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/13.0/web_many2one_simple/static/img/preview.gif
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+Attributes:
+~~~~~~~~~~~
+
+* regex [String]: Defines the regex to test the search criteria
+* can_create [Boolean]: Allow create new records
+* can_write [Boolean]: Allow edit the linked record
+
+Options:
+~~~~~~~~
+
+* search [Dictionary]
+  * field: The field to use in the search process [Default is 'id']
+  * oper: The operator (like, ilike, etc...) [Default is '=']
+
+* no_create: Invalidates the 'can_create' flag
+* no_write: Invalidates the 'can_write' flag
+* noOpen: Disables open the linked record
```

Comment:
Why this one is CamelCase instead of snake_case?

Reply:
Follows Odoo implementation

Reply:
Really can be used in both ways:
- noOpen
- no_open

I updated the readme to use the snake case style